core/src/doc/lives.rs

```diff
@@ -53,44 +53,25 @@ impl<'a> Document<'a> {
 					true => &self.initial,
 					false => &self.current,
 				};
-				// Ensure that a session exists on the LIVE query
-				let sess = match lv.session.as_ref() {
-					Some(v) => v,
-					None => continue,
-				};
-				// Ensure that auth info exists on the LIVE query
-				let auth = match lv.auth.clone() {
-					Some(v) => v,
+
+				let mut lqctx = match lv.context(ctx) {
+					Some(ctx) => ctx,
 					None => continue,
 				};
-				// We need to create a new context which we will
```

Comment:
Why is this being deleted, it's necessary
Auth is required
Transactions are required

Reply:
I've seen it's moved to the context function. Could we please rename `context` to `construct_lq_context`, as just having `context` can be confused with some sort of global context function